tools/nnpackage_tool/model2nnpkg/README.md

```diff
@@ -7,17 +7,47 @@ It takes `modelfile` as input and generates `nnpackage`.
 ## Usage
 
 ```
-Usage: model2nnpkg.sh [options] modelfile
-Convert modelfile to nnpackage.
+Usage: model2nnpkg.sh [options]
```

Comment:
Here is my opinion. It is just for showing.

```suggestion
Usage: model2nnpkg [options]
```

1. Replace all `model2nnpkg.sh` to `model2nnpkg`, and remove `model2nnpkg.sh` and `model2nnpkg.py` from README.md.
2. Introduce `model2nnpkg` (which just executes either `model2nnpkg.sh` or `model2nnpkg.py` internally) OR
Rename `model2nnpkg.py` to `model2nnpkg`.
3. For backward compatibility, keep `model2nnpkg.sh` for a while